crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -391,10 +391,16 @@ def proposal_update
           flash[:notice] = answer[1] if answer[0] >= 300 and answer[0] < 400
           flash[:notice] = t('barclamp.proposal_show.commit_proposal_success') if answer[0] == 200
           if answer[0] == 202
-            flash_msg = answer[1].map { |node_dns|
-                 NodeObject.find_node_by_name(node_dns).alias
-            }.join ", "
-            flash[:notice] = "#{t('barclamp.proposal_show.commit_proposal_queued')}: #{flash_msg}"
+            missing_nodes = answer[1].map { |node_dns| NodeObject.find_node_by_name(node_dns) }
+
+            unready_nodes = missing_nodes.map(&:alias).join(", ")
+            unallocated_nodes = missing_nodes.reject(&:allocated).map(&:alias).join(", ")
```

Comment:
Yeah, I'm mostly thinking that the duplication is what could (possibly) be confusing (because of too much information). I don't have a very strong feeling about this, though. So if anyone else feels the current way is fine, we can just keep it.